docs/content/connection-options.md

```diff
@@ -148,6 +148,13 @@ These are the other options that MySqlConnector supports.  They are set to sensi
     <td>True to have MySqlDataReader.GetValue() and MySqlDataReader.GetDateTime() return DateTime.MinValue for date or datetime columns that have disallowed values.</td>
   </tr>
   <tr>
+    <td>Keep Alive, Keepalive</td>
+    <td>0</td>
+    <td>TCP Keepalive idle time.  A value of 0 indicates that the OS Default keepalive settings are used.
```

Comment:
Connector/Net documents "A value of 0 indicates that keepalive is not used."; do we want to be different by default?